static/js/lightbox_canvas.js

```diff
@@ -133,6 +139,26 @@ var LightboxCanvas = (function () {
                 }
             });
 
+            // event listener to handle zoom in and out from using keyboard keys z/Z and +/-
+            // in the canvas
+            // these hotkeys are not implemented in static/js/hotkey.js as the code in
+            // static/js/lightbox_canvas.js and static/js/lightbox.js isn't written a way
+            // that the LightboxCanvas instance created in lightbox.js can be
+            // accessed from hotkey.js. Major code refactoring is required in lightbox.js
+            // to implement these keyboard shortcuts in hotkey.js
+            document.addEventListener('keydown', function (e) {
+                if (e.key === "Z" || e.key === '+') {
+                    funcs.setZoom(meta, '+');
+                    funcs.displayImage(canvas, context, meta);
+                } else if (e.key === "z" || e.key === '-') {
+                    funcs.setZoom(meta, '-');
+                    funcs.displayImage(canvas, context, meta);
+                }
+                e.preventDefault();
+                e.stopPropagation();
+            });
```

Comment:
This key handler fires even if the lightbox isn't open.  You're going to want to add a check for `overlays.lightbox_open()` or something before you do anything at the top of this function.